src/pages/search/advanced_search/AdvancedSearchPage.tsx

```diff
@@ -50,10 +61,12 @@ export const AdvancedSearchPage = () => {
   const categoryShould = (params.get(ResultParam.CategoryShould)?.split(',') as PublicationInstanceType[] | null) ?? [];
   const topLevelOrganizationId = params.get(ResultParam.TopLevelOrganization);
   const unitId = params.get(ResultParam.Unit);
+  const excludeSubunits = params.get(ResultParam.ExcludeSubunits) === 'true' ? 'true' : null;
```

Comment:
Oppnår ikke noe med å "caste" `'true'` til `'true'`. Bare fint om vi får denne til å bli riktig type raskere

```suggestion
  const excludeSubunits = params.get(ResultParam.ExcludeSubunits) === 'true';
```

Reply:
Da kan vi under her ha 
```
excludeSubunits,
topLevelOrganization: excludeSubunits && !!unitId ? null : topLevelOrganizationId,
```